courses/requirements/wharton.py

```diff
@@ -55,15 +61,47 @@ def _clean_data(data):
     'SS': 'Social Science',
     'NSME': 'Natural Sciences, Math, and Engineering',
     'FGE': 'Uncategorized / Flex Gen Ed Only',
+    'TIA': 'Technology, Innovation, and Analytics',
+    'GEBS': 'Global Economy, Business, and Society'
 }
 
 
+GEBS_URL = 'https://undergrad-inside.wharton.upenn.edu/requirements-2017/global-economy-business-society/'
+TIA_URL = 'https://undergrad-inside.wharton.upenn.edu/requirements-2017/technology-innovation-analytics/'
+
+
+def _get_curriculum_page(url):
+    resp = requests.get(url)
+    resp.raise_for_status()
+
+    reqs = []
+    soup = BeautifulSoup(resp.content.decode('utf-8'), features='html.parser')
+    for lst in soup.select('.wpb_wrapper ul.bullet-list'):
+        for item in lst.select('li'):
+            match = re.match(r'([A-Z]{0,4})\s?(\d{3})\s?:', item.text, re.I)
+            if match is not None:
+                res = match.groups()
+                reqs.append({
+                    'department': res[0],
+                    'course_id': int(res[1]),
```

Comment:
I don't know if we want to cast to an int here -- a course like `ECON-001` will get represented as `department: ECON, course_id: 1` in the system, even though we would want the full three-character code `001`.

Reply:
Fixed! Nice catch!